lib/src/main/java/edu/utexas/tacc/tapis/files/lib/clients/GlobusDataClient.java

```diff
@@ -705,9 +701,22 @@ private FileInfo buildGlobusRootFileInfo(String path)
   {
     FileInfo fileInfo = new FileInfo();
     fileInfo.setName(path);
-    fileInfo.setType(FileInfo.FILETYPE_DIR);
+    fileInfo.setType(FileInfo.FileType.FILETYPE_DIR);
     fileInfo.setPath("/~/");
     return fileInfo;
   }
 
```

Comment:
@scblack321 I think this should be equivalent to the old code, but maybe you could take a look and double check it for globus?

Reply:
Yes, this looks good.